packages/nimble-components/src/rich-text/mention-list-box/styles.ts

```diff
@@ -0,0 +1,11 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+
+export const styles = css`
+    ${display('inline-flex')}
+
+    nimble-listbox {
+        max-height: 168px;
```

Comment:
This should probably be a token. Or maybe a better option is to use something like `calc(5.5 * ${controlHeight})` to indicate that we want to show at most 5.5 options. (There was a recommendation from UX that we use X.5 so that it is immediately obvious that there are more items scrolled out of view.)

Reply:
Updated as per the suggestion.